### PR TITLE
Fix existing tests

### PR DESCRIPTION
**Pull request by nikitakovaliov92**

### CODE REVIEW

**Comment by nikitakovaliov92** [comment]
`passed tests:` https://github.com/veraPDF/veraPDF-rest/actions/runs/10123932618

File: tests/rest_api/tests/api_validate/test_api_validate_details.py [hunk 1]
```diff
@@ -6,7 +6,7 @@
     ApiValidateDetails,
 )
 
-details_version = re.compile(r"\d\.\d\d\.\d\d\d")
+details_version = re.compile(r"\d\.\d\d\.\d\d")
```
Comment (MaximPlusov) [inline tests/rest_api/tests/api_validate/test_api_validate_details.py:9]: Should be r"\d\.\d\d\.\d+"

Comment (nikitakovaliov92) [reply tests/rest_api/tests/api_validate/test_api_validate_details.py]: fixed

Comment (nikitakovaliov92) [reply tests/rest_api/tests/api_validate/test_api_validate_details.py]: `passed tests:` https://github.com/veraPDF/veraPDF-rest/actions/runs/10134160731

**Review by MaximPlusov — APPROVED** [review]

**Pull request merged**